forta-tornado-cash-starter-kit/src/agent.spec.js

```diff
@@ -54,5 +57,44 @@ describe("TornadoCash contract interactions", () => {
         }),
       ]);
     });
+
+    it("should not return a finding if the address that was funded from TornadoCash created a contract", async () => {
+      mockTxEvent.filterLog.mockReturnValue([
+        {
+          args: {
+            to: "0xa",
+          },
+        },
+      ]);
+
+      mockTxEvent.transaction = {
+        from: "0xa",
+        to: "", // contract creation
+        data: "0x1234567Test",
+      };
+      const findings = await handleTransaction(mockTxEvent);
+
+      expect(findings).toStrictEqual([]);
+    });
+
+    it("should not return a finding if the address that was funded from TornadoCash interacted with a TornadoCash contract", async () => {
+      mockTxEvent.filterLog.mockReturnValue([
+        {
+          args: {
+            to: "0xa",
+          },
+        },
+      ]);
+
+      mockTxEvent.transaction = {
+        from: "0xa",
+        to: "0xbB93e510BbCD0B7beb5A853875f9eC60275CF498", // TC contract
```

Comment:
```suggestion
        to: "0xbB93e510BbCD0B7beb5A853875f9eC60275CF498", // Ethereum 10 WBTC TC contract
```
Suggestion to included which contract for easier checking.